.github/workflows/accessibility_test.yml

```diff
@@ -0,0 +1,131 @@
+name: Accessibility test
+
+on: pull_request
+
+jobs:
+  run-accessibility-test:
+    strategy:
+      matrix:
+        # Run against the current branch (represented by '', which will use the default ref when using the checkout action) and the base branch
+        branch: [ '', '${{ github.base_ref }}' ]
+
+    # Uses if/else expression evaluation workaround from https://github.com/actions/runner/issues/409#issuecomment-752775072
+    # 'matrix.branch == github.base_ref' is the condition, 'base' is the true value and 'current' is the false value
+    name: Accessibility test (${{ matrix.branch == github.base_ref && 'base' || 'current' }} branch)
+  
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres:13.3
```

Comment:
I didn't provide DockerHub credentials for the postgres and elasticsearch image, but the CircleCI config does:
https://github.com/dockstore/dockstore-ui2/blob/5fc1ed6a35e8be101189fca3cd0f127689355099/.circleci/config.yml#L18-L20

I assume it was added because of rate-limiting? Do we need it here, and if so, do these credentials already exist in our GitHub secrets (I can't see them)?

Reply:
I'd hope that github has some form of Docker image caching and/or a similar deal.